### ISSUE TITLE
[FEATURE REQUEST] Trakt Integration

### ISSUE THREAD

**Issue by DickenSerm** [IgnisDa/ryot#875]
I would like to see Trakt integration.

I have several services that are all connected to Trakt.tv. Such as Emby.

People have already asked about Emby, but unfortunately that is not possible without a premium account. Trakt.tv would therefore be very good as a bridge.

**Comment by Ardakilic** [comment]
+1 , trakt.tv integration would have been great. I also sync my Jellyfin watch history to trakt, and it's the current hub for my tracking history.

**Comment by elvis0288** [comment]
is this something that you will implement in the near future? this is totally a plus for me as i use trakt for everything right now and having to do something twice will be complicated.

**Comment by IgnisDa** [comment]
Would this be something that is required if #68 gets implemented?

**Comment by elvis0288** [comment]
well they are 2 different things. but in my case to completely get rid of trakt i guess i need statistics and others things like an android app so having ryot been able to import to ryot in the meantime, everytime i change something on trakt would be nice to have

**Comment by IgnisDa** [comment]
What would an android app do?

**Comment by elvis0288** [comment]
actually nothing but having widgets to quickly complete shows and movies would be really nice, even though having everything doing it for you with the scrobber would be enough

**Comment by IgnisDa** [comment]
Makes sense. In case you didn't notice though, Ryot can be installed as a PWA to your android device.

**Comment by elvis0288** [comment]
yes, i have not try that yet because i need to have it on the reverse proxy which i have not done yet

**Comment by IgnisDa** [comment]
I think once #68 is implememted, i will close this since I not want to add trakt integration.
